bundles/catalogue/metadataflyout/view/MetadataPage.js

```diff
@@ -326,7 +335,16 @@ Oskari.clazz.define('Oskari.catalogue.bundle.metadataflyout.view.MetadataPage',
             // );
 
             return replacedText;
+        },
+
+        /**
+         * @method showErrorMessage
+         * Render 'metadata not found' message to ui
+         */
+        _showMetadataNotFoundMessage: function () {
+            jQuery(this.ui[0]).text(this.locale.notFound);
```

Comment:
`this.ui` is already a jQuery object. Could we just call `.text()` on that directly without making a new jQuery object?